ci-operator/config/openshift/release/openshift-release-master__nightly-4.13.yaml

```
@@ -659,6 +659,16 @@ tests:
       enable:
       - observers-resource-watch
     workflow: openshift-e2e-aws-sdn
+- as: e2e-aws-localzone-ovn
+  interval: 168h
+  steps:
+    cluster_profile: aws-2
```

@rvanderp3 can we change as we had the confirmation?:

```suggestion
    cluster_profile: aws-3
```